### DISCUSSION TITLE
GUI doesn't open on Windows Headless Server

### DISCUSSION THREAD

**Discussion by sparksandshadows** [apohl79/audiogridder#550] [General]
I've got a remote computer with no monitor. The GUI simply doesn't draw, and instead freezes up the DAW. This is true of every version. Any plans for a permanent fix for this?

**Comment by jaacMusic** [comment]
I have dealt with this issue. You will need to either put a dummy display connector to your video card or install a virtual display adapter that allows you to run your PC without a physical video card but resolves the resolution issue. Sorry, I don't remember which virtual display driver I use but I did find a free one online using Google for search.

**Comment by jaacMusic** [comment]
I have dealt with this issue. You will need to either put a dummy display connector to your video card or install a virtual display adapter that allows you to run your PC without a physical video card but resolves the resolution issue. Sorry, I don't remember which virtual display driver I use but I did find a free one online using Google for search.

**Reply by dhstsw** [reply]
On unraid it uses its virtual display by default (wich is a VNC server).
In that case the problem is that it uses the same port as ag, so it doesn't solve the situation.

**Reply by apohl79** [reply]
You can increase the server id to run the server on a different port. The server binds to 55056+id.

**Reply by dhstsw** [reply]
i'll try that.
thx

**Reply by dhstsw** [reply]
Works perfectly!
Thanks!

**Reply by sparksandshadows** [reply]
TightVNC is not working for this case.. does anyone have a good recommendation?
